docs/topics/class-based-views/intro.txt

```diff
@@ -279,6 +279,13 @@ that it can be used on an instance method. For example::
         def dispatch(self, *args, **kwargs):
             return super(ProtectedView, self).dispatch(*args, **kwargs)
 
+Or, more succinctly, you can decorate the class instead, and pass the name
+of the method to be decorated as the keyword argument ``name``::
+
+    @method_decorator(login_required, name="dispatch")
```

Comment:
Very minor nitpick: double quotes are used here, and then single quotes are used two lines down. Single quotes would be consistent with the previous example in this section.